src/pathfinding_service/cli.py

```diff
@@ -47,6 +47,13 @@
     help="Number of block confirmations to wait for",
 )
 @click.option("--enable-debug", default=False, is_flag=True, hidden=True)
+@click.option("--operator", default="John Doe", type=str, help="Name of the service operator")
+@click.option(
+    "--info-message",
+    default="This is your favorite pfs for token network registry ",
```

Comment:
I'd put this into a constant and use that in the `info` endpoint for the comparison as well

Reply:
ok